### DISCUSSION TITLE
Can't install CLI on macbook

### DISCUSSION THREAD

**Discussion by ghost** [deta/cloud-docs#533] [Help]
I write in terminal 👉 curl -fsSL https://get.deta.dev/cli.sh | sh

installation is in progress and everything seems to be fine but ...
at the next step deta login the terminal writes an error command not found

**Comment by abdelhai** [comment]
please, try restarting the terminal and report back.

**Reply by ghost** [reply]
how can i restart the terminal ?

**Reply by lemonyte** [reply]
@maureryakov quit the Terminal app and reopen it, or source your profile file with `source ~/.zshrc` (more info [here](https://stackoverflow.com/a/68351246) if you are using a different shell). Or just reboot your Mac.

**Reply by ghost** [reply]
I rebooted my macbook this way, it did not lead to anything. Anyway, after installing the CLI, the terminal writes command not found.

**Reply by ghost** [reply]
Do I understand correctly that I download applications on DETA according to the same principle as on HEROKU?

**Reply by ghost** [reply]
![Снимок экрана 2022-10-18 в 21 33 04](https://user-images.githubusercontent.com/73276809/196515395-1a6768a2-fdb8-4e6a-9f73-fbc67b38c8fc.png)

**Comment by rareseu** [comment]
@maureryakov That happened to me but I managed to fix it by adding the deta cli manually in .bash_profile.

1. Go to your Home Folder (/Users/**x**, in my case /Users/rares)
2. If you can't see the hidden folders, press **CMD** + **Shift** + **.** (DOT)
3. Go to .deta/bin/ (/Users/x/.deta/bin)
4. Press right click on the **deta** executable
5. Press ALT, and press "Copy `deta` as PathName"
6. Go back to your Home Folder (/Users/**x**, in my case /Users/rares)
7. Open .bash_profile file and create an alias with the pathname copied at step 5, as following:
`alias deta=/Users/x/.deta/bin/deta` where x it's your user name, in my case rares
8. Save and restart terminal

**Reply by ghost** [reply]
It worked, but now the process does not open in safari and asks for another browser , why?

**Reply by rareseu** [reply]
It opens your default browser. For example, I am using Brave as default browser, when I input deta login, a new Brave tab/window opens. Cheers!

**Reply by ghost** [reply]
I realized that the default browser opens, but safari does not work, and I will not install another browser to use deta. In general, I did not like the deta service. thank you all for your help, you are awesome!

**Comment by bimberman** [comment]
@rareseu I am using an M1 Macbook and I do not have a .bash_profile file in my /Users/x directory. I tried updating the .zshrc file but it did not work. Can you please advise?

**Reply by rareseu** [reply]
@bimberman That's ok! That means you haven't had aliases added yet!
Just create a file **.bash_profile** in your Users folder. ex: /Users/USERNAME

You can simply create it with Terminal too: open it, type **cd**, type **pwd**, make sure you're in /Users/USERNAME and then **touch .bash_profile**

**Reply by bimberman** [reply]
Thank you.
